src/language.jsx

```diff
@@ -59,7 +59,10 @@ const apiDataTableNames = {
 }
 
 const buttons = {
+  cancel: 'Cancel',
   close: 'Close',
+  confirm: 'Confirm',
+  saveChanges: 'Save Changes',
 }
```

Comment:
I think you made the buttons category in another ticket. I agree this text is generic enough to reusable, so good call to put it here.